src/trace/implementations/rhh.rs

```diff
@@ -97,7 +98,8 @@ mod val_batch {
     #[derive(Abomonation, Debug)]
     pub struct RhhValStorage<L: Layout> 
     where 
-        <L::Target as Update>::Key: Default + HashOrdered
+        <L::Target as Update>::Key: HashOrdered,
+        <L::Target as Update>::KeyOwned: Default + HashOrdered,
```

Comment:
This probably shouldn't be a requirement of the structure itself, but rather the implementations that need to push data in (`Default` so we can pad locations, and `HashOrdered` for actual owned insertions).